packages/frontend/app/src/session/Sessions.tsx

```diff
@@ -13,7 +13,10 @@ interface SessionsProps {
 }
 
 const Sessions: FC<SessionsProps> = args => {
```

Comment:
I really do wish this was props, not args can we maybe put this in this PR since we are modifying this component.